src/memoryprofiler.js

```diff
@@ -525,13 +524,13 @@ var emscriptenMemoryProfiler = {
     self.drawContext.fillRect(0, 0, self.canvas.width, self.canvas.height);
 
     self.drawContext.fillStyle = "#FF8080";
-    self.fillLine(STACK_BASE, STACK_MAX);
+    self.fillLine(stackMax, stackBase);
```

Comment:
these look flipped, shouldn't it be `base, max`?

Reply:
I think these should go from low to high.. so I think maybe there was an existing bug... it looked fine when I tested it but I should go back and check what it looks like in reverse  I guess.

Reply:
Yeah I think it was wrong before.. fillLine definitely look like it expects low to high..